### PR TITLE
✨ update completion circle

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

File: src/components/CompleteOMeter.js [hunk 1]
```diff
@@ -2,76 +2,37 @@ import * as React from 'react';
...
-import { get } from 'lodash';
 
-import { ROLES } from 'common/constants';
-
-const CompleteOMeter = compose(withTheme)(({ theme, role, percentage, className }) => (
+const CompleteOMeter = compose(withTheme)(({ theme, percentage, className }) => (
   <div
-    css={`
-      width: 310px;
+    className={css`
```
Comment (alex-wilmer) [inline src/components/CompleteOMeter.js]: why the change to `className` ?

File: src/components/UserProfile/CompletionWrapper.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import { compose } from 'recompose';
+import { injectState } from 'freactal/lib/inject';
+import { VictoryPie } from 'victory-pie';
+
+const enhance = compose(withTheme, injectState);
+
+const CompletionWrapper = ({
+  completed,
+  theme,
+  state: { loggedInUser },
```
Comment (alex-wilmer) [inline src/components/UserProfile/CompletionWrapper.js]: can this be abstracted a bit? maybe it doesn't need to be, but it feels weird for this particular component to care specifically about user state

Comment (alex-wilmer) [reply src/components/UserProfile/CompletionWrapper.js]: looks like you're not using it actually

File: src/components/CompleteOMeter.js (hunk 1 above)
Comment (Jephuff) [reply src/components/CompleteOMeter.js]: ... I don't know hahaha. oops

File: src/components/UserProfile/CompletionWrapper.js (hunk 2 above)
Comment (Jephuff) [reply src/components/UserProfile/CompletionWrapper.js]: hmm yeah, it was there  when I had the profile image in there as well.  removed now!

any idea why eslint isn't warning on unused destructured arguments? Does atom warn you?

File: src/components/CompleteOMeter.js (hunk 1 above)
Comment (Jephuff) [reply src/components/CompleteOMeter.js]: fixed :)
